templates/flat-black.php

```diff
@@ -7,20 +7,20 @@
 
 		<div id="list">
 			<?php
-			$total = count( $songs );
-			for ( $i = 0; $i < $total; $i++ ) {
```

Comment:
Now that it is changed to foreach, The `//end for` comment isn't valid anymore. It should be `//end foreach` instead. that is on line 32.